src/Paket.Core/RuntimeGraph.fs

```diff
@@ -0,0 +1,185 @@
+﻿namespace Paket
+
+open Domain
```

Comment:
@matthid can you add some doccoms for the functions in this file? I already know i'll have forgotten what half of this does in a few days